boards/PDM/Test/Src/Test_StateMachine.cpp

```diff
@@ -176,3 +240,9 @@ TEST_F(PdmStateMachineTest, check_air_closed_state_is_broadcasted_over_can)
         CANMSGS_PDM_STATE_MACHINE_STATE_AIR_CLOSED_CHOICE,
         App_CanTx_GetPeriodicSignal_STATE(can_tx_interface));
 }
+
+TEST_F(
+    PdmStateMachineTest,
+    check_vbat_voltage_is_broadcasted_over_can_in_all_states)
+{
```

Comment:
Plenty of unit tests for  stting `can_tx` signals, to be added later once everything else looks good